DataFormats/HcalDetId/interface/HcalDetId.h

```diff
@@ -38,6 +38,8 @@ class HcalDetId : public DetId {
   int iphi() const { return id_&0x7F; }
   /// get the tower depth
   int depth() const { return (id_>>14)&0x1F; }
+  /// get the tower depth
+  uint32_t maskDepth() const { return (id_ | 0x1C000); }
```

Comment:
@bsunanda 
I see many bit masks here, appearing as "magic": can this become `static constexpr` members?
@Dr15Jones please advise.